frontend/src/pages/class/index.tsx

```diff
@@ -13,24 +14,42 @@ const messages = defineMessages({
   },
 });
 
+const StyledContainer = styled(Container)`
+  padding-top: 0;
+  padding-left: 0;
+`;
+
+const PageDescription = styled.p`
+  font-size: 1rem;
+  color: #000000ff;
+  margin-top: 1rem;
+  margin-bottom: 3rem;
+`;
+
 const ListClasses = () => {
   return (
     <>
       <Header title={messages.title} />
-      <Container>
+      <StyledContainer>
         <Breadcrumbs>
           <CrtNavigation breadcrumb />
         </Breadcrumbs>
-        <PageHeader>
+        <Heading mt="10" mb="2">
+          <FormattedMessage
+            id="ListClasses.pageTitle"
```

Comment:
The `pageTitle` ID may be easily confused with the `title` ID (defined above in `messages`)
`header` here seems more accurate, since this is where it's displayed.